src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -241,6 +241,12 @@ public override void Execute(DocumentsOperationContext context)
                             cmd.Document.TryGet(Constants.Documents.Metadata.Key, out metadata);
                             LastEtag = putResult.Etag;
 
+                        
+                            metadata.Modifications = new DynamicJsonValue(metadata)
```

Comment:
I removed the other fields, and kept them in the metadata.
And we want to do it this way to the client doesn't have to re-blit the metadata after receiving it.